src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -74,6 +80,8 @@ public EventType(final String name, final String topic, final String owningAppli
         this.schema = schema;
         this.defaultStatistic = defaultStatistic;
         this.options = options;
+        this.writeScope = writeScope;
+        this.readScope = readScope;
```

Comment:
They are optional, I think they'd better be singular because when you do operation on optional you get the lambda and there you have to use plural because you have a set.